.github/workflows/tag-release-drafter.yml

```diff
@@ -0,0 +1,34 @@
+name: Tag Release drafter
+on:
+  push:
+    branches:
+      - main
```

Comment:
I think we also want to checkout to main right? Since we're tagging the actual commit we are releasing: aka the merge commit.

Reply:
Does the simple checkout not checkout the merge commit? Did you mean we want to checkout the HEAD of the PR?

I did testing in my own private repo and it seems like the tagging is correctly tagging the merge commit.

Reply:
we want to tag the merge commit. We can test out if checkout actually does what we want for sure

Reply:
sounds good.